disttask/importinto/dispatcher.go

```diff
@@ -194,10 +195,12 @@ func (dsp *ImportDispatcherExt) unregisterTask(ctx context.Context, task *proto.
 // OnNextSubtasksBatch generate batch of next stage's plan.
 func (dsp *ImportDispatcherExt) OnNextSubtasksBatch(ctx context.Context, taskHandle dispatcher.TaskHandle, gTask *proto.Task) (
 	resSubtaskMeta [][]byte, err error) {
+	nextStep := dsp.GetNextStep(gTask)
 	logger := logutil.BgLogger().With(
 		zap.String("type", gTask.Type),
 		zap.Int64("task-id", gTask.ID),
-		zap.String("step", stepStr(gTask.Step)),
+		zap.String("curr-step", stepStr(gTask.Step)),
```

Comment:
In master code, I have added `nextStep` as a parameter of `OnNextSubtasksBatch`. 😂 My consideration is, `GetNextStep` can be called once while `OnNextSubtasksBatch` maybe called for multiple time for a subtask

Reply:
i say that `GetNextStep` now has a `handle` param, we can remove it, and move those logic into `OnNextSubtasksBatch`, if there's no need to do merge-sort, just return 0 subtask metas, and let dispatcher call it again on next step.

Reply:
If `GetNextStep` is heavy to compute, it's better we let framework call it once and send the result to the argument of `OnNextSubtasksBatch`

> if there's no need to do merge-sort, just return 0 subtask metas

Currently I think it's also acceptable. In future we may have a complex step transition and must use non-linear steps.